CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## [0.29.3](https://github.com/XAMPPRocky/octocrab/compare/v0.29.2...v0.29.3) - 2023-08-15
+- add associated_check_runs to CommitHandler
```

Comment:
You don't need to add anything to CHANGELOG, we have an automation that does that for us.

Reply:
Reverted this file.